modules/specifying-regions-zones-infrastructure-vsphere.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+// * post_installation_configuration/sphere-failure-domain-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="specifying-regions-zones-infrastructure-vsphere_{context}"]
+= Specifying multiple regions and zones for your cluster on vSphere
+
+You can configure the `infrastructures.config.openshift.io` configuration resource to specify multiple regions and zones for your {product-title} cluster that runs on a VMware vSphere instance.
+
+Topology-aware features for the cloud controller manager and the vSphere Container Storage Interface (CSI) Operator Driver require information about the vSphere topology where you host your {product-title} cluster. This topology information exists in the `infrastructures.config.openshift.io` configuration resource. 
+
+Before you specify regions and zones for your cluster, you must ensure that all datacenters and compute clusters contain tags, so that the cloud provider can add labels to your node. For example, if `datacenter-1` represents `region-a` and `compute-cluster-1` represents `zone-1`, the cloud provider adds an `openshift-region` category label with a value of `region-a` to `datacenter-1`.  Additionally, the cloud provider adds an `openshift-zone` category tag with a value of `zone-1` to `compute-cluster-1`.
+
+[NOTE]
+====
+You can migrate control plane nodes with vMotion capabilities to a failure domain. After you add these nodes to a failure domain, the cloud provider adds `topology.kubernetes.io/zone` and `topology.kubernetes.io/region` labels to these nodes.  
+====
+
+.Prerequisites
+* You created the `openshift-region` and `openshift-zone` tag categories on the vCenter server.
+* Optional: If you defined *API* and *Ingress* static IP addresses to the installation program, you must ensure that all regions and zones share a common layer 2 network. This configuration ensures that API and Ingress Virtual IP (VIP) addresses can interact with your cluster.
+* You ensured that each datacenter and compute cluster contains tags that represent the name of their associated region or zone, or both.
```

Comment:
I recommend swapping this line with the Optional line above, unless this is optional as well - then I recommend adding `Optional:` to the start of the line as you did above.